arbnode/batch_poster.go

```diff
@@ -1494,7 +1512,13 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 	// In theory, this might reduce gas usage, but only by a factor that's already
 	// accounted for in `config.ExtraBatchGas`, as that same factor can appear if a user
 	// posts a new delayed message that we didn't see while gas estimating.
-	gasLimit, err := b.estimateGas(ctx, sequencerMsg, lastPotentialMsg.DelayedMessagesRead, data, kzgBlobs, nonce, accessList, delayProof)
+	var delayedMsgOverride uint64
```

Comment:
I could also use an explanation of why delayedMsgOverride is being set here.

Reply:
Should be clearer now (I hope)